runtime/autoload/provider/clipboard.vim

```diff
@@ -175,13 +173,23 @@ function! s:clipboard.set(lines, regtype, reg) abort
     call jobsend(jobid, a:lines)
     call jobclose(jobid, 'stdin')
     let selection.owner = jobid
+    let ret = 1
   else
     echohl WarningMsg
     echomsg 'clipboard: failed to execute: '.(s:copy[a:reg])
     echohl None
-    return 0
+    let ret = 1
+  endif
+
+  " The previous provider instance should exit when the new one takes
+  " ownership, but ensure it gets kill to not fill up the job table.
```

Comment:
there is a typo. But the old comment was "active voice" which is almost always better.

```suggestion
  " ownership, but kill it to be sure we don't fill up the job table.
```